Sources/GRInputField/Common/InputFieldTraits.swift

```diff
@@ -29,7 +31,8 @@ public struct InputFieldTraits: Sendable {
         numpadReturnKeyTitle: String? = nil,
         clearButtonMode: UITextField.ViewMode = .whileEditing,
         isSecureTextEntry: Bool = false,
-        isHapticsAllowed: Bool = true
+        isHapticsAllowed: Bool = true,
```

Comment:
V tomto prípade keďže nejde o getter haptics stavu ale konštruktor by som to nazval allowHaptics: Bool, čo nesedí asi tea s tým isSecure entry čo je škoda.